CMakeLists.txt

```diff
@@ -283,7 +283,7 @@ option(FLB_FILTER_GEOIP2               "Enable geoip2 filter"
 option(FLB_FILTER_NIGHTFALL            "Enable Nightfall filter"                      Yes)
 option(FLB_FILTER_WASM                 "Enable WASM filter"                           Yes)
 option(FLB_PROCESSOR_LABELS            "Enable metrics label manipulation processor"  Yes)
-option(FLB_PROCESSOR_ATTRIBUTES        "Enable atributes manipulation processor"      Yes)
```

Comment:
Not sure the FLB_PROCESSOR_ATTRIBUTES line is left in master....

Reply:
my bad